tox.ini

```diff
@@ -26,11 +26,6 @@ passenv =
     XAUTHORITY
     NUMPY_EXPERIMENTAL_ARRAY_FUNCTION
     PYVISTA_OFF_SCREEN
-conda_deps =
-    # use conda to install numcodecs on mac py3.9
-    py39-macos: numcodecs
-conda_channels =
```

Comment:
Why do conda_deps and conda_channels need to be removed here? I don't quite understand how that is related to ruff.

Reply:
I thinkt hat it comes from this part of description:

>  Also dropped some old Conda cruft from tox.ini which wasn't used.

Reply:
No need for it to be a separate PR, just checking that you intended this 😄